### ISSUE TITLE
[WebJobs.Extensions - v1.0.1+] TimerTrigger does not resolve tokenized [TimerTrigger(expression)] values 

### ISSUE THREAD

**Issue by JoeBrockhaus** [Azure/azure-webjobs-sdk-extensions#138]
using `[TimerTrigger("%ConfigNameToResolve%")] TimerInfo timerInfo` with an appropriate `INameResolver` registered in the `JobHostConfiguration`, the JobHost fails to startup as an exception is thrown when the Function on which the trigger is defined is indexed. 

The exception I'm receiving is that the `expression` is not a valid TimeSpan. 
Looking at [the latest available source](https://github.com/Azure/azure-webjobs-sdk-extensions/blob/master/src/WebJobs.Extensions/Extensions/Timers/Scheduling/TimerSchedule.cs#L54), it appears this should not be possible, as the `TimerSchedule` is created using a factory that uses the resolver. 

Indeed, after decompiling the 1.0.1 version of the WebJobs.Extensions NuGet, regressions were introduced into the `TimerTriggerAttribute`. The code no longer attempts to resolve the expression using the configured `INameResolver`.

```
        public TimerTriggerAttribute(string expression)
        {
            CronSchedule cronSchedule = null;
            if (!CronSchedule.TryCreate(expression, out cronSchedule))
            {
                TimeSpan timeSpan = TimeSpan.Parse(expression);
                this.Schedule = new ConstantSchedule(timeSpan);
                this.UseMonitor = timeSpan.TotalMinutes >= 1;
                return;
            }
            this.Schedule = cronSchedule;
            if ((int)cronSchedule.InnerSchedule.GetNextOccurrences(DateTime.Now, DateTime.Now + TimeSpan.FromMinutes(1)).ToArray<DateTime>().Length > 1)
            {
                this.UseMonitor = false;
                return;
            }
            this.UseMonitor = true;
        }
```

**Comment by mathewc** [comment]
INameResolver support was added **after** v1.0.1 we released (in [this commit](https://github.com/Azure/azure-webjobs-sdk-extensions/commit/10d7d4c2b14dc555b81ff4059c656cca94163c86)). It is available in the 2.0.0-beta1 prerelease versions on nuget/myget.

**Comment by JoeBrockhaus** [comment]
I just came back to this and realized latest dev is at 2.0.0-beta1. 
You beat me to it :D
